dependency-manager-tests/xcconfigs/unit-tests-target.xcconfig

```diff
@@ -0,0 +1,2 @@
+#include "app-target.xcconfig"
+PROVISIONING_PROFILE_SPECIFIER[sdk=iphoneos*]=// overwrite to empty, so Xcode picks "Automatic"
```

Comment:
what does `overwrite to empty` mean? 
• should we leave it empty? or
• Xcode picks `automatic` if we leave it empty so that we should provide a value here?

Reply:
Here we need to overwrite it with empty string, so it doesn't use the setting from `app-target.xcconfig`. Then Xcode uses it's default value (auto). I tried overwriting it with `"auto"`, `"Auto"` and `"Automatic"`, but none of them worked.

Alternatively, we could split it into two xcconfigs:
```
// app-target.xcconfig
DEVELOPMENT_TEAM[sdk=iphoneos*]=xxx
CODE_SIGN_IDENTITY[sdk=iphoneos*]=yyy
PROVISIONING_PROFILE_SPECIFIER[sdk=iphoneos*]=zzz
```
```
// unit-tests-target.xcconfig
DEVELOPMENT_TEAM[sdk=iphoneos*]=xxx
CODE_SIGN_IDENTITY[sdk=iphoneos*]=yyy
```
But this would requires more hassle and two separate `datadog-app-target.xcconfig` and `datadog-unit-tests-target.xcconfig`. All will duplicate parts of the setup. Current approach optimizes number of duplication.

Reply:
okay i see 👍 
but i think the current comment message is not clear enough.

_this line overrides `app-target.xcconfig` and sets prov profile to `Auto` in its target_
what do you think?

Reply:
Will do 👍

Reply:
Done ✅